schemas/exchange/min-meta/json-min-db-sequence_clusters-col-cluster_provenance.json

```diff
@@ -863,6 +863,19 @@
                      }
                   ]
                },
+               "pdbx_reference_DOI": {
```

Comment:
Could you, please, provide more context for this new attribute?

Reply:
@valasatava This was recently added to the PDBx/mmCIF dictionary: https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v5_next.dic/Items/_software.pdbx_reference_DOI.html

You can see that in the revision description for version 5.411: https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v5_next.dic/Index/

Please let me know if you want this to be suppressed.

Reply:
@valasatava I have suppressed this attribute in the schemas.